app/modules/intelligence/agents_copy/plan.py

```
@@ -0,0 +1,145 @@
+
+
+""" ------------------------------------------------------------- """
+""" Agents Services """
+""" ------------------------------------------------------------- """
+
+
+# this can be reused in many places (including conversations etc)
+class AgentsPostgresRepo:
+    """Repository for agents, get list create update delete etc"""
+
+    def __init__(self, db: Session):
```

Ruff (0.8.2)

Undefined name `Session`

(F821)